extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -386,19 +375,19 @@ public void setClaims(Map<String, String> claims) {
         }
 
         /**
-         * CredentialsProvider which provides a client secret
+         * CredentialsProvider, which provides a client secret.
          */
         @ConfigGroup
         public static class Provider {
 
             /**
-             * The CredentialsProvider name which should only be set if more than one CredentialsProvider is registered
+             * Optional. The CredentialsProvider name, which should only be set if more than one CredentialsProvider is registered
```

Comment:
@sergey
By "only" do we mean: "You must set the name if more than one CredentialsProvider is registered. Otherwise, you must not set this value."